dcc-submission-ui/dictionary/app/index.html

```diff
@@ -41,9 +42,19 @@
         <script src="bower_components/regex-colorizer/regex-colorizer.js"></script>
         <script src="bower_components/d3-tip/index.js"></script>
         <script src="bower_components/jsoneditor/dist/jsoneditor.js"></script>
+        <script src="bower_components/jsondiffpatch/public/build/jsondiffpatch-full.min.js"></script>
 
-        <script src="scripts/vendor/beautify.js"></script>
+        <script>
+            var jsondiffpatch = jsondiffpatch.create({
```

Comment:
Maybe do this instead:

https://github.com/fuzzyyeti/todojitsu/blob/7564baded0ae954d6bd6f51fc7e263f2e2415e59/public/javascripts/services/diffpatcher.js#L2